engine/api/api.go

```diff
@@ -127,6 +128,14 @@ type Configuration struct {
 			Password string `toml:"password"`
 		} `toml:"kafka"`
 	} `toml:"events" comment:"#######################\n CDS Events Settings \n######################"`
+	FeaturesFlipping struct {
+		Izanami struct {
+			ApiURL       string `toml:"api_url"`
+			ClientID     string `toml:"client_id"`
+			ClientSecret string `toml:"client_secret"`
+			Token        string `toml;"token" comment:"Token shared between Izanami and CDS to be able to send webhooks from izanami"`
+		}
```

Comment:
missing `toml:"izanami" + comment about what is is, why use it, etc... or just 'enable alpha feature, use it with precaution'

Reply:
and something under https://ovh.github.io/cds/hosting/ about izanami?

Reply:
not for now, it's not mandatory. We also have to discuss about release ;)